node/database/migrations/20190508153151-Users.js

```diff
@@ -24,6 +24,12 @@ module.exports = {
           type: Sequelize.BOOLEAN,
         })
       })
+      .then(() => {
```

Comment:
It's better to create new migration instead of editing old ones. Migrations are like git history in a sense, it's the history of database changes and if history is edited, then problems may appear. Also migrations should always be reversable, like taking steps back in history. This is why you should always be able to see directly from migrations what was edited in db schema and in which order.

Reply:
Created new migration